app/components/AddTokenForm/index.js

```diff
@@ -0,0 +1,50 @@
+/**
+*
+* AddTokenForm
```

Comment:
This is a component, not just a function. It doesn't add a token form, it *is* a token form. Please rename appropriately.